worker/src/request.rs

```diff
@@ -37,7 +37,7 @@ impl From<web_sys::Request> for Request {
                     }
                     u
                 }),
-            headers: Headers(req.headers()),
+            headers: req.headers().into(),
```

Comment:
It's unfortunate that this is a breaking API change, but I suppose the original API was very brittle.